README.TXT

```diff
@@ -1,3 +1,32 @@
+Ultimate Tapan Kaikki
+=====================
+
+This is the legendary 1990's game Ultimate Tapan Kaikki, ported to
+SDL2. It runs at least on macOS, Linux, FreeBSD and Windows.
+
+Suomipelit GitHub: https://github.com/suomipelit

```

Comment:
Could mention what this organization is about.

Reply:
Any suggestions what to put there? :)

Reply:
Done.